Code/Tests/MutaliskViewer/Makefile

```diff
@@ -56,20 +56,21 @@ SRCS=\
 	TimelinePlayer.cpp\
 	vram.c\
 
+CXX_FLAGS:=$(CC_FLAGS)\
+	-std=c++11\
+	-fexceptions\

```

Comment:
Both of these are not necessary, and in particular, enabling exception is not right way to mimic the behavior of gcc410